src/build/app/mod.rs

```diff
@@ -2121,12 +2142,47 @@ impl<'help> App<'help> {
     ///     .unwrap_or_else(|e| e.exit());
     /// ```
     /// [`App::try_get_matches_from`]: App::try_get_matches_from()
+    ///
+    /// ### Panics
+    ///
+    /// If both [`AppSettings::Multicall`]
+    /// and [`AppSettings::NoBinaryName`] are used.
```

Comment:
I believe we generally put checks like this in `src/build/app/debug_asserts.rs` and not have the functions check and panic

Reply:
Thanks for the tip, I was basing adding panics on the last line of `CONTRIBUTING.md`, could do with a pointer from there.

Reply:
https://github.com/clap-rs/clap/pull/2818